### DISCUSSION TITLE
Noted collector (GENERAL) [UR]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#10] [Card Balance]
Digital card game :

Noted Collector (GENERAL) [UR]
INFLUENCE
If your Warrior with an UNIQUE Attachment leaves play, the UNIQUE Attachment is returned to your hand.

This card is currently useless.  

My suggestion :

INFLUENCE
Opponent cannot take control of your unique cards.

It could help players against "a greater need".
"Heist" cannot be played on your unique attachment. :laughing:

**Comment by Kite80** [comment]
I like the idea that unique equipment can't be stolen while Noted Collector in in play, but it doesn't fit much with warriors

**Comment by TomekPL-1** [comment]
I like 21-ric idea for Noted collector. This will defenitely make this card more popular.

If it is not to OP I would add to this "cannot take control or discard UNIQUE cards"

**Comment by 21-ric** [comment]
@TomekPL-1 if  we add "cannot discard Unique cards".
"Noted collector" will protect your unique cards against :
-fool's gold
-leveled field
-supply line severed
-malfunction
-pacifism
-cleansing flame
and too many cards.

I think it's a nice idea but it's too powerfull.

**Reply by Kite80** [reply]
That's why we should stick to equipments and exclude warriors and support. Noted Collector refers to relics after all. Btw leveld field should still work, since it would discard everything except warriors, Noted collector would go into discard pile together with relics

**Reply by 21-ric** [reply]
So, what is your suggestion?
